editor/src/clj/editor/code_completion.clj

```diff
@@ -197,7 +217,7 @@
                  ;; :snippet, :text, :text_esc and :err are unreachable because
                  ;; they only exist on the top level, while content-string is
                  ;; used to build string for choices, variables and placeholders
-                 :rule (content-string sb (first children))
+                 :syntax_rule (content-string sb (first children))
```

Comment:
I had to rename `rule` to `syntax_rule`, otherwise antlr compiler complains that rule is a keyword...